src/pages/about.astro

```diff
@@ -1,15 +1,30 @@
 ---
 import Layout from '../layouts/Layout.astro'
 import Hero from '../components/elements/Hero.svelte'
+import ContentImageFull from '../components/sections/ContentImageFull.svelte'
+import FooterFull from '../components/footer/FooterFull.svelte'
+import RedButton from '../components/buttons/WhiteButton.svelte'
+import MetricCard from '../components/cards/MetricCard.svelte'
+import { getCollection, type CollectionEntry } from 'astro:content'
+// interface Props {
+//   entry: CollectionEntry<'aboutMetrics'>
+// }
 
 import '../styles/global.css'
-import { imageConfig } from 'astro:assets'
-import ContentImageFull from '../components/sections/ContentImageFull.svelte'
 
-import FooterFull from '../components/footer/FooterFull.svelte'
+// export async function getStaticPaths() {
```

Comment:
Recommend removing the commented out sections - it will just cause confusion later.

Reply:
good call thanks will do